eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/EidasSamlParserApplication.java

```diff
@@ -128,14 +129,4 @@ protected void configure() {
             }
         });
     }
-
-    private MetatronProxy buildMetatronProxy(EidasSamlParserConfiguration configuration, Environment environment) {
```

Comment:
This method was okay to keep - it is useful but rename it to `createMetatronProxy ` (like we have for other things)